src/batch-client/index.ts

```diff
@@ -192,6 +192,16 @@ function updateAbsoluteFolderPath(
 		return folder;
 	});
 }
+/**
+ * Return an empty string in case it's empty array or null value,
+ * else return an Array.
+ *
+ * @param value An Array or empty String
```

Comment:
graphql would expect single data type to be returned, how are we handling case when we return string instead of array?

Reply:
We convert it only when we send data to Server. I have updated code comments. We are doing so, because, for Server, following values means:

`[]` - No change to reflect on server.
`''` - Set value to empty for field.

Reply:
Added comment as discussed.